components/functions-library-mgt/org.wso2.carbon.identity.functions.library.mgt/src/main/java/org/wso2/carbon/identity/functions/library/mgt/util/FunctionLibraryManagementConstants.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.functions.library.mgt.util;
+
+/**
+ * Function Library Management constant class.
+ */
+public class FunctionLibraryManagementConstants {
+
+    /**
+     * Enum for error messages.
+     */
+    public enum ErrorMessage {
+
+        ERROR_CODE_UNEXPECTED("SCL-65001", "Unexpected Error"),
+        ERROR_CODE_DATABASE_CONNECTION("SCL-65001", "Couldn't get a database connection."),
+        ERROR_CODE_VALIDATE_SCRIPT_LIBRARY_SCRIPT("SCL-60002", "Script library script of %s contains errors."),
+        ERROR_CODE_REQUIRE_SCRIPT_LIBRARY_NAME("SCL-60003", "Script library name is required"),
+        ERROR_CODE_REQUIRE_SCRIPT_LIBRARY_SCRIPT("SCL-60003", "Script library script is required"),
+        ERROR_CODE_ALL_READY_EXIST_SCRIPT_LIBRARY("SCL-65004", "Already a script library available with the name: %s."),
```

Comment:
Typo.. change ALL_READY to ALREADY

Reply:
[Fixed commit](https://github.com/wso2/carbon-identity-framework/pull/2679/commits/3d6fa4d553b0d78ca83f8d85f66d01572a985143)